waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -1,7 +1,7 @@
 {.push raises: [Defect].}
 
 import
-  std/[sequtils, tables, times, streams, os, deques],
+  std/[sequtils, tables, times, streams, os, deques, bitops],
```

Comment:
Where the `bitops` module is used?

Reply:
Addressed in [a42eb55](https://github.com/status-im/nwaku/pull/1318/commits/a42eb55c4a70df3d310471c58d810a4b262028cf), and removed `streams` as well.